### ISSUE TITLE
Official DEB not working on Ubuntu 20 LTS

### ISSUE THREAD

**Issue by rolodoom** [davy7125/polyphone#104]
Official DEB is not working on Ubuntu 20 LTS
Need library libstk-4.5.0 and Ubuntu 20  this library has been updated to libstk-4.6.1

**Comment by mirabilos** [comment]
I’m currently working on packaging Polyphone for Debian officially. I will make it available in my PPA for *buntu 20.04 LTS when successful.

**Comment by mirabilos** [comment]
[My PPA](https://launchpad.net/~mirabilos/+archive/ubuntu/ppa/) has a polyphone package for Ubuntu 20.04 LTS. It’s a backport (20.04 is ooooooold…) of the one I just uploaded to Debian. It’s the **only** legal Polyphone binary distribution in existence and currently excludes SF3 support.

Testing feedback is, of course, welcome.

**Comment by davy7125** [comment]
Dear Mirabilos, first thank you very much for the help you have given in making a package that passed the Debian quality rules. This is a task I tried some years before and then I gave up, my priorities been elsewhere... So I really appreciate!

Regarding this ticket, I added a package here for Ubuntu 20.04 (included the sf3 support... I know it's bad!)
https://www.polyphone-soundfonts.com/download

And more and more I will give links to the repositories.
